dataLayer/modulemanager.datalayer.json

```diff
@@ -1,2 +1,17 @@
-{
-}
+MMdataLayer = [{
+    'languageIsoCode': 'en',
+    'countryIsoCode ': 'US',
+    'originAirportIataCodes ': 'MIA,TXL,SXF',
+    'destinationAirportIataCodes': 'LHR,MIL,LAX',
+    'fareClass': 'ECONOMY',
+    'journeyType': 'ONE_WAY',
+    'isLoggedIn': 'TRUE',
```

Comment:
Note that for booleans, we just populate true or false in lowercase without quotation marks following again JSON best practices.